celery/bin/worker.py

```diff
@@ -152,7 +189,6 @@ def detach(path, argv, logfile=None, pidfile=None, uid=None,
 @click.option('--prefetch-multiplier',
               type=int,
               metavar="<prefetch multiplier>",
-              callback=lambda ctx, _, value: value or ctx.obj.app.conf.worker_prefetch_multiplier,
```

Comment:
This cannot be removed. The default value must be derived from the app if it is not supplied.